index.html

```diff
@@ -3563,6 +3538,301 @@ <h2>Privacy</h2>
+					<dd>
+						<dl class="variablelist">
+							<dt><a href="https://www.w3.org/TR/html/sections.html#the-nav-element"
+								><code>nav</code></a></dt>
+							<dd>
+								<p>A <code>role</code> attribute with the value <code>doc-toc</code> is REQUIRED</p>
+								<p>In this order:</p>
+								<ul class="nomark">
+									<li><a href="https://www.w3.org/TR/html/dom.html#heading-content"><code>HTML Heading
+												content</code></a>
+										<code>[0 or 1]</code></li>
+									<li>
+										<code>ol</code> or <code>ul</code>
+										<code>[exactly 1]</code>
+									</li>
+								</ul>
+							</dd>
+							<dt><a href="https://www.w3.org/TR/html/grouping-content.html#the-ol-element"
+										><code>ol</code></a> and <a
+									href="https://www.w3.org/TR/html/grouping-content.html#the-ul-element"
+										><code>ul</code></a></dt>
+							<dd>
+								<ul class="nomark">
+									<li>
+										<p>
+											<code>li</code>
+											<code>[1 or more]</code></p>
+									</li>
+								</ul>
+							</dd>
+							<dt><a href="https://www.w3.org/TR/html/grouping-content.html#the-li-element"
+										><code>li</code></a></dt>
+							<dd>
+								<p>In this order:</p>
+								<ul class="nomark">
+									<li>
+										<p><code>a</code>
+											<code>[exactly 1]</code></p>
+									</li>
+									<li>
+										<p>
+											<code>ol</code> or <code>ul</code>
+											<code>[0 or 1]</code></p>
+									</li>
+								</ul>
+							</dd>
+							<dt><a href="https://www.w3.org/TR/html/textlevel-semantics.html#the-a-element"
+										><code>a</code></a></dt>
+							<dd>
+								<p>In any order:</p>
+								<ul class="nomark">
+									<li>
+										<p>
+											<a href="https://www.w3.org/TR/html/dom.html#phrasing-content"><code>HTML
+													Phrasing content</code></a>
+											<code>[1 or more]</code></p>
+									</li>
+								</ul>
+							</dd>
+						</dl>
+						<p>Note that there are no restrictions on the attributes allowed on these elements.</p>
+					</dd>
+				</dl>
+
+				<p>The content model of the <code>nav</code> element is interpreted as follows:</p>
+
+				<ul class="conformance-list">
+					<li>
+						<p>The first <code>ol</code> or <code>ul</code> descendant of the <code>nav</code> element
+							defines the set of table of contents links, and its child list item elements represent the
+							primary level of content navigation.</p>
+					</li>
+					<li>
+						<p>Each list item identifies a heading, structure or other item of interest. A child
+								<code>a</code> element identifies and/or describes the target location.</p>
+					</li>
+					<li>
+						<p>The <code>a</code> element MUST provide a non-zero length text label after concatenation of
+							all child content and application of white space normalization rules. If a meaningful label
+							cannot be constructed from the text content of the <code>a</code> element &#8212; for
```

Comment:
Doesn't contradict, no, as you can have a non-zero label that is unintelligible if it depends on visuals, embeded content, etc. But I think we can merge. Either a non-zero label or an aria-label, and also an aria-label when the simple concatenation of the text content would lead to an incomplete representation of the link.